internal-client/src/main/java/org/apache/uniffle/client/request/RssDecommissionRequest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.uniffle.client.request;
+
+public class RssDecommissionRequest {
+  private boolean on;
```

Comment:
Could you give this configuration option a better name?

Reply:
I have not good idea.

Reply:
This class will be used in both decommission and cancel decommission.  `on=true` means decommission and `on=false` means cancel decommission.

Reply:
Should we separate it to two rpc methods?

Reply:
It is OK for me.

Reply:
Why do we need to cancel decomission? Is there for other systems the function?

Reply:
> Why do we need to cancel decomission? Is there for other systems the function?

This function is required in case of incorrect operation or plan change. HDFS supports cancel decomission too.

Reply:
I didn't see it in the HDFS. HDFS's decomission api is called `refreshNodes`.

Reply:
> I didn't see it in the HDFS. HDFS's decomission api is called `refreshNodes`.

Yes, we add datanode to `exclude-file` and then `refreshNodes` if we need to decomission it . And we remove it from `exclude-file` and then `refreshNodes` if we need to cancel decomission it.

Reply:
Maybe it's better that we change `on` to `cancel`.

Reply:
It's ok for me too.

Reply:
> It's ok for me too.

Let us use `cancel` to replace `on`. `On` don't give me enough information.

Reply:
Updated